doc/man3/SSL_read_early_data.pod

```diff
@@ -186,6 +199,20 @@ In the event that the current maximum early data setting for the server is
 different to that originally specified in a session that a client is resuming
 with then the lower of the two values will apply.
 
+Some server applications may wish to have more control over whether early data
+is accepted or not. For example to mitigate replay risks (see L</REPLAY PROTECTION>
+below) or to decline early_data when the server is heavily loaded. The functions
```

Comment:
I think this "For example..." is not a complete sentence.  Maybe join the two with ", for example,"?

Reply:
Fixed.